packages/playground/remote/service-worker.ts

```diff
@@ -6,13 +6,32 @@ import { getURLScope, removeURLScope } from '@php-wasm/scopes';
 import { applyRewriteRules } from '@php-wasm/universal';
 import {
 	awaitReply,
-	convertFetchEventToPHPRequest,
+	convertFetchEventToPHPRequest as convertFetchEvent,
 	initializeServiceWorker,
 	cloneRequest,
 	broadcastMessageExpectReply,
 } from '@php-wasm/web-service-worker';
 import { wordPressRewriteRules } from '@wp-playground/wordpress';
 
+async function convertFetchEventToPHPRequest(event: any) {
```

Comment:
Why is this needed? Why not make the changes in  `@php-wasm/web-service-worker`?